woudc_data_registry/models.py

```diff
@@ -1184,6 +1372,14 @@ def sync(ctx):
         click.echo('{}...'.format(plural_caps))
 
         registry_contents = registry_.query_full_index(clazz)
+        models_dict_key = ""
+
+        for index in registry_contents:
+            models_dict_key = str(type(index))[7:-1]
+            break
+
+        models_dict.update({models_dict_key: registry_contents})
```

Comment:
Easier to do `models_dict[models_dict_key] = registry_contents, people are more used to that way as well and find it easier to read.